tools/image_processing/imagej2/imagej2_bunwarpj_align.xml

```diff
@@ -1,75 +1,140 @@
-<?xml version='1.0' encoding='UTF-8'?>
 <tool id="imagej2_bunwarpj_align" name="Align two images" version="@WRAPPER_VERSION@.0">
     <description>with bUnwarpJ</description>
     <macros>
         <import>imagej2_macros.xml</import>
     </macros>
-    <expand macro="fiji_requirements" />
-    <command>
-<![CDATA[
-    python $__tool_directory__/imagej2_bunwarpj_align.py
-    --target_image "$target_image"
-    --target_image_format $target_image.ext
-    --source_image "$source_image"
-    --source_image_format $source_image.ext
-    #if $align_with_mask_cond.align_with_mask == 'yes':
-        --target_mask "$target_mask"
-        --target_mask_format $target_mask.ext
-        --source_mask "$source_mask"
-        --source_mask_format $source_mask.ext
-    #end if
-    --min_scale_def $min_scale_def
-    --max_scale_def $max_scale_def
-    --max_subsamp_fact $max_subsamp_fact
-    --divergence_weight $divergence_weight
-    --curl_weight $curl_weight
-    --image_weight $image_weight
-    --consistency_weight $consistency_weight
-    #if $advanced_options_cond.advanced_options == 'landmarks':
-        --landmarks_weight $advanced_options_cond.landmarks_weight
-        --landmarks_file "$advanced_options_cond.landmarks_file"
-    #else if $advanced_options_cond.advanced_options == 'affine':
-        --target_affine_file "$advanced_options_cond.target_affine_file"
-        --source_affine_file "$advanced_options_cond.source_affine_file"
-    #else if $advanced_options_cond.advanced_options == 'mono':
-        --mono "true"
-    #else if $advanced_options_cond.advanced_options == 'save_transformation':
-        --target_trans_out "$target_trans_out"
-        --source_trans_out "$source_trans_out"
-    #end if
-    --source_out "$source_out"
-    --source_out_datatype $source_out_datatype
-    #if $advanced_options_cond.advanced_options != 'mono':
-        --target_out "$target_out"
-        --target_out_datatype $target_out_datatype
-    #end if
-    --jython_script $__tool_directory__/imagej2_bunwarpj_align_jython_script.py
-]]>
-    </command>
+    <expand macro="fiji_requirements"/>
+    <command detect_errors="exit_code"><![CDATA[
+#import os
+#import tempfile
```

Comment:
That's not going to work, this creates files in a random place on the host, this won't be available for containerized executors or most standard setups with compute nodes that don't run Galaxy.

Reply:
How do we create an absolute path to send to Java so that is doesn't prepend a "/" to a relative file on disk?

Reply:
I've eliminated the use of temple - no longer needed since the Python scripts have been eliminated.  Thanks @mvdbeek !